ding2.install

```diff
@@ -271,3 +271,22 @@ function ding2_update_7008() {
 function ding2_update_7009() {
   module_enable(array('redirect'));
 }
+
+/**
+ * Disable the cookiecontrol module.
+ * Enable and configure the eu-cookie-compliance module.
+ */
+function ding2_update_7010() {
+  if (module_exists('cookiecontrol')) {
+    module_disable(array('cookiecontrol'));
+    drupal_uninstall_modules(array('cookiecontrol'));
+    variable_del('cookiecontrol_privacynode');
+    variable_del('cookiecontrol_text');
+    variable_del('cookiecontrol_fulltext');
+  }
+  module_enable(array('eu_cookie_compliance'));
+  ding2_set_cookie_page();
```

Comment:
Have you considered whether this should delete any preexisting cookie information page?